marklogic-data-hub-central/ui/src/components/flows/flows.tsx

```diff
@@ -256,6 +258,7 @@ const Flows: React.FC<Props> = (props) => {
                                         <div
                                             className={styles.run}
                                             aria-label={'runStep-' + stepNumber}
+                                            data-testid={'runStep-' + stepNumber}
```

Comment:
can we try using getByLabelText. If it works then we might not need data-testid

Reply:
@bsrikan I originally tried with getByLabelText, but it didn't seem to find it that way.

Reply:
ok.